src/excmds.ts

```diff
@@ -473,9 +473,18 @@ export async function fixamo() {
  * **Be *seriously* careful with this: you can use it to open any URL you can open in the Firefox address bar.**
  *
  * You've been warned.
+ *
+ * This uses the [[browser]] setting to know which binary to call. If you need to pass additional arguments to firefox (e.g. '--new-window'), make sure they appear before the url.
  */
 //#background
-export async function nativeopen(url: string, ...firefoxArgs: string[]) {
+export async function nativeopen(...args: string[]) {
+    const index = args.findIndex(arg => !arg.startsWith("-"))
```

Comment:
Does this stop URLs with dashes from being opened?

Reply:
`firefox -url` doesn't work and neither does `:nativeopen -url`. `firefox http://-url` works and so does `:nativeopen`. I believe this PR doesn't break anything :).